sdks/python/apache_beam/runners/interactive/pipeline_fragment.py

```diff
@@ -100,17 +100,23 @@ def deduce_fragment(self):
         self._runner_pipeline.runner,
         self._options)
 
-  def run(self, display_pipeline_graph=False, use_cache=True):
+  def run(self,
+          display_pipeline_graph=False,
+          use_cache=True,
+          blocking_run=False):
```

Comment:
Use `blocking` instead of `blocking_run`? (Since that is what we use in other places.)

Reply:
Renamed `blocking_run` to `blocking`.
Renamed all `try-finally` preserve-and-reset variables with `preserved_` prefix.